client/src/pages/demo/components/index.tsx

```diff
@@ -22,11 +23,9 @@ const DemoComponent: React.FunctionComponent = () => {
         <hr className="w-3/4 mx-auto" />
 
         <div className="p-4 border">
-          <h1>Component: Iframe</h1>
-          <br />
           <Iframe
             src="https://www.youtube.com/embed/HGl75kurxok"
-            className="border"
+            className={''}
```

Comment:
- i'm not sure. whats the changes for.
- please double check _"file changes"_ tab in your PR to fix possible overlooked changes.